client/admin_client.go

```diff
@@ -0,0 +1,49 @@
+package client
+
+import (
+	"time"
+
+	"github.com/cozy/cozy-stack/client/request"
+	"github.com/cozy/cozy-stack/pkg/config/config"
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/tlsclient"
+)
+
+type AdminClient struct {
+	Client
+}
+
+func (ac *AdminClient) NewInstanceClient(domain string, scopes ...string) (*Client, error) {
+	// For the CLI client, we rely on the admin APIs to generate a CLI token.
+	// We may want in the future to rely on OAuth to handle the permissions with
+	// more granularity.
```

Comment:
I don't understand this comment. Can you explain me what are you thinking about?

Reply:
I just copy/pasted the comment from https://github.com/cozy/cozy-stack/blob/master/cmd/root.go#L55-L58 since I copy/pasted the logic.
This might not be relevant anymore though.

Reply:
Hu, yes, let's remove this comment